.github/workflows/main.yml

```diff
@@ -155,10 +153,17 @@ jobs:
       - name: Running REST API and SDK tests
         id: run_tests
         run: |
-          pip3 install --user '/tmp/cvat_sdk/[pytorch]'
-          pip3 install --user cvat-cli/
-          pip3 install --user -r tests/python/requirements.txt
-          pytest tests/python/ -s -v
+          pip3 install -r cvat-sdk/gen/requirements.txt
+          ./cvat-sdk/gen/generate.sh
+
+          pip3 install -r ./tests/python/requirements.txt
+          pip3 install -e ./cvat-sdk
+          pip3 install -e ./cvat-cli
+
+          pytest tests/python/ --cov --cov-report xml
```

Comment:
I ran this command locally and I got XML file that says that coverage is 0%. Do you know what the problem could be?

Reply:
And I also see 0% in codecov: https://app.codecov.io/gh/opencv/cvat/tree/ay%2Fadd-codecov

Reply:
Fixed